remotes/docker/authorizer.go

```diff
@@ -148,9 +148,11 @@ func (a *dockerAuthorizer) AddResponses(ctx context.Context, responses []*http.R
 	defer a.mu.Unlock()
 	for _, c := range auth.ParseAuthHeader(last.Header) {
 		if c.Scheme == auth.BearerAuth {
-			if err := invalidAuthorization(c, responses); err != nil {
+			if retry, err := invalidAuthorization(c, responses); err != nil {
```

Comment:
Will one retry with no delay between be sufficient to get over the issue?

Reply:
I don't thin so, as the method name `invalidAuthorization` hinted, the registry server has found the auth invalid, retry with the same token is useless.

Reply:
@mmerkes Have I explained your question? this pr still need your approvement.